### ISSUE TITLE
Matplotlib .plot() throws error "Procedure entry point mkl_aa_fw_init_workdivision not found in mkl_core.dll"

### ISSUE THREAD

**Issue by tdbuck78** [ContinuumIO/anaconda-issues#8327]
Windows 7 64-bit  |  Python 3.6.3  | Anaconda custom (64-bit) | NumPy 1.13.3 | Matplotlib 2.1.2

Trying to run the following code crashes jupyter kernel and shows the error:

`import matplotlib.pyplot as plt`
`%matplotlib inline`
`import numpy as np`
`x = np.linspace(0,5,11)`
`y = x**2`
`plt.plot(x,y)`

"Procedure entry point mkl_aa_fw_init_workdivision not found in mkl_core.dll"

Already tried reinstalling Anaconda + matplotlib.

**Comment by BagsySigel** [comment]
Did you fix this? Have the same error.

**Comment by msarahan** [comment]
It looks like you probably have an older MKL version that's getting found before ours does.  Look in C:\Windows\system32.  I recommend removing these files, but you may break whatever software put them there.  If you can figure out what software put them there, then move them into the folder with that software.

Putting 3rd party DLLs into the system32 folder is very evil, because it affects things so globally, and puts you in "DLL hell" - where you are now.

The rules for how DLLs get found on Windows are at https://msdn.microsoft.com/en-us/library/7d83bc18.aspx

**Comment by tdbuck78** [comment]
@BagsySigel Yeah I did. Unfortunately I had to reinstall Windows, since this is a DLL issue.
